framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -428,6 +447,45 @@ private AnnotationMirror getAnnotationMirrorOfMetaAnnotation(
                 result.add(new Postcondition(expr, postcondAnno, anno));
             }
         }
+
+        return result;
+    }
+
+    /**
+     * Returns the set of repeatable postconditions.
```

Comment:
What set?

Reply:
I meant to say that it returns a collection of repeatable annotations.
Would the below line would be correct?
```Returns repeatable annotations```

Reply:
I have updated the methods and documentation. Please give a review.